healthcareai/common/transformers.py

```diff
@@ -19,17 +22,82 @@ def fit(self, X, y=None):
         self.obj_list = X.select_dtypes(include=['object']).columns.values
 
         self.fill = pd.Series([X[c].value_counts().index[0]
-            if X[c].dtype == np.dtype('O') else X[c].mean() for c in X],
-            index=X.columns)
+                               if X[c].dtype == np.dtype('O') else X[c].mean() for c in X], index=X.columns)
 
         # return self for scikit compatibility
         return self
 
     def transform(self, X, y=None):
-        X = X.fillna(self.fill)
+        result = X.fillna(self.fill)
 
         for i in self.obj_list:
-            X[i] = X[i].astype(object)
+            result[i] = result[i].astype(object)
+
+        return result
+
+
+class DataFrameConvertTargetToBinary(TransformerMixin):
+    """
+    Convert predicted col to 0/1 (otherwise won't work with GridSearchCV)
```

Comment:
The R package can only handle 'Y' or 'N' in the target column anyways. Let's just make sure it's well documented.